pkgs/development/interpreters/elixir/generic-builder.nix

```diff
@@ -22,6 +22,7 @@ in
 
     buildInputs = [ erlang rebar makeWrapper ];
 
+    LOCALE_ARCHIVE = "${pkgs.glibcLocales}/lib/locale/locale-archive";
```

Comment:
`glibcLocales` is linux only

Reply:
Any ideas on a solution which would work both on Linux and on Mac osx?

Reply:
I don't think it's necessary there

Reply:
Build without this line: http://ix.io/AHh
Build with this line: http://ix.io/AHl

It definitely removes the warnings. I also inserted calls to `locale` in preBuild step and without this line the `locale` command were printout out warnings too. Can you suggest an alternative solution to get rid of the warnings?

Reply:
You can make add conditionally depending on the platform using `stdenv.isLinux`.

Reply:
@gleber sorry for the confusion, I was just suggesting to make it conditional with eg. `stdenv.lib.optionalString stdenv.isLinux` to avoid breaking the darwin build

Reply:
Done!